app/pages/sections/consignee/ConsigneeExportPage.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.sections.consignee
+
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object ConsigneeExportPage extends QuestionPage[Boolean] {
+
+  override def path: JsPath = JsPath \ toString
```

Comment:
It might be useful to start including a section object in the path. So that when we look at the data in mongo it will also be saved by section.

So something like `JsPath \ "consignee" \ toString`

This would probably also help when we come to having to work out section complete logic for the task list page in the future.

Reply:
@MJCallahanPage Pushed https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/35/commits/e59e3a220f575aef12413a6409af8a4b4fbace56 to address this change.